src/events/get-all-augur-logs.js

```diff
@@ -51,8 +51,8 @@ function getAllAugurLogs(p, batchCallback, finalCallback) {
           try {
             return parseLogMessage(contractName, eventName, log, eventsAbi[contractName][eventName].inputs);
           } catch (exc) {
-            console.error("parseLogMessage error", exc);
             console.log(contractName, eventName, log, eventsAbi[contractName], chunkOfBlocks);
+            throw new Error(exc);
```

Comment:
This is within an async callback; we have a `finalCallback`, perhaps we should pass an error back to it here? Until we get this turned into an async function, catching this error might be hard